### DISCUSSION TITLE
Add support for ZITADEL OIDC

### DISCUSSION THREAD

**Discussion by bitfactory-sem-denbroeder** [langfuse/langfuse#1715] [Ideas]
### Describe the feature or potential improvement

Zitadel is an open-source SSO implementation, it works nativly with auth,js and seems to be a rather small integration.

**Comment by marcklingen** [comment]
Hi @bitfactory-sem-denbroeder, open for PRs on this one. You can check how Okta/Auth0 are added throughout the repo.
